packages/web-react/src/components/FileUploader/useFileQueue.ts

```diff
@@ -17,18 +17,23 @@ export const useFileQueue = (): FileQueueReturn => {
     return queue;
   };
 
-  const addToQueueHandler = (key: string, file: File) => {
-    setQueue((prev) => new Map(prev.set(key, file)));
+  const addToQueueHandler = (key: string, file: File, meta?: FileQueueValueMetaType) => {
+    setQueue((prev) => {
+      const newValue: FileQueueValueType = { file };
+      if (typeof meta !== 'undefined') newValue.meta = meta;
```

Comment:
```suggestion
      if (meta != null) {newValue.meta = meta;}
```
I think we can just ask if the meta is null or undefined or is there a problem I cannot see?

Reply:
you are right, changed